apps/web/src/pages/quick-start/components/layout/FooterLayout.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Grid } from '@mantine/core';
+import styled from '@emotion/styled';
+
+interface IFooterLayoutProps {
+  leftSide: React.ReactNode;
+  rightSide: React.ReactNode;
+}
+
+export function FooterLayout({ leftSide, rightSide }: IFooterLayoutProps) {
+  return (
+    <FooterWrapper>
+      <Grid justify="space-between" style={{ width: '100%', padding: '0 80px' }} gutter={0}>
+        <Grid.Col span={4}>
+          <LeftSide>{leftSide}</LeftSide>
+        </Grid.Col>
+        <Grid.Col span={4}>
+          <MiddleSide> .. </MiddleSide>
```

Comment:
The 'navigation dots' will be added later once Pawel will create his component and I will adjust it to my need if needed.